rl4co/envs/__init__.py

```diff
@@ -18,6 +18,10 @@
     SPCTSPEnv,
     TSPEnv,
     MDCPDPEnv,
+    VRPLEnv,
+    OVRPEnv,
+    VRPTWEnv,
```

Comment:
Right, since they are the same environment, we could have a simple option to pass the data generator as a user choice, but should not be different classes